commons-codec/1.14.0.wso2v1/pom.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~ Copyright (c) 2005-2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Is this license header format correct?

Reply:
@gayashanbc

Reply:
Fixed in https://github.com/wso2/orbit/pull/434/commits/1d8bb7042d700c2f86c1823b16024d2d9befa7a3